src/NUnitFramework/nunitlite.runner/nunitlite.runner-netcore/Runner/AutoRun.cs

```diff
@@ -0,0 +1,86 @@
+﻿// ***********************************************************************
```

Comment:
Let's try this and see if it may be a better way to go for other runners as well.